artifactory/commands/buildinfo/publish.go

```diff
@@ -91,6 +93,35 @@ func (bpc *BuildPublishCommand) Run() error {
 	if err != nil {
 		return err
 	}
+
+	buildTime, err := time.Parse(buildinfo.TimeFormat, buildInfo.Started)
+	if errorutils.CheckError(err) != nil {
+		return err
+	}
+
+	// Convert from nanoseconds to milliseconds
+	timestamp := buildTime.UnixNano() / 1000000
```

Comment:
The calculated ```timestamp``` is not needed in the case if Artifactory 6. Let's therefore redesign the code, so that it is calculated only if needed.